gh_actions_exporter/types.py

```diff
@@ -54,8 +55,24 @@ class Repository(BaseModel):
     visibility: str
 
 
+class Organization(BaseModel):
+    login: str
+
+
 class WebHook(BaseModel):
     workflow_run: Optional[WorkflowRun] = None
     workflow_job: Optional[WorkflowJob] = None
     repository: Optional[Repository] = None
+    organization: Optional[Organization] = None
     zen: Optional[str] = None
+
+class JobCost(BaseModel):
+    name: str
+    cost: int
+
+class CheckRunData(BaseModel):
+    summary: str
+    settings: dict
+    workflow_run: dict
+    jobs_cost: list[dict]
+    total_cost: float
```

Comment:
All fields set as dict are not typed enough, what's inside the dict?

Reply:
Oh ok, I thought putting just `dict` was enough, I'll change for every occurence I can find